tests/gateway/stripe/test_stripe.py

```diff
@@ -350,10 +350,13 @@ def test_list_customer_sources(sandbox_gateway_config):
 def test_get_client(gateway_config):
     assert _get_client(**gateway_config.connection_params).api_key == "secret"
 
-
+'''
+FIXME
+The current implementation uses this method to pass the Stripe public key to
+Stripe.js in the template. I don't think that's what was intended!
```

Comment:
You changed the logic of the `get_client_token` so you should change this test to expect a new response